shopinvader_schema_address/schemas.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from extendable_pydantic import ExtendableModelMeta
+from pydantic import BaseModel
+
+from odoo.addons.pydantic import utils
+
+
+class State(BaseModel, metaclass=ExtendableModelMeta):
+    id: int
+    name: str
+    code: str
+
+    class Config:
+        orm_mode = True
+        getter_dict = utils.GenericOdooGetter
+
+
+class Country(BaseModel, metaclass=ExtendableModelMeta):
+    id: int
+    name: str
+    code: str
+
+    class Config:
+        orm_mode = True
+        getter_dict = utils.GenericOdooGetter
+
+
+class Address(BaseModel, metaclass=ExtendableModelMeta):
+    id: int
+    name: str
```

Comment:
I think `name`, `street`, ... must not be required.
An address is a `res.partner` record hence fields are not required on record.
It induces a problem in cart services : a cart has a `partner_shipping_id` (which is by default equal to `partner_id`) and this partner may have no address.

Reply:
Okay, I will change this